src/sgame/sg_spawn.cpp

```diff
@@ -1026,6 +1026,18 @@ static void InitDisabledItemCvars()
 	G_SpawnStringIntoCVar( "disabledBuildables", g_disabledBuildables );
 }
 
+static void InitTacticBehaviorsCvar()
+{
+	static Cvar::Callback<Cvar::Cvar<std::string>> g_tacticBehaviors(
+	   "g_tacticBehaviors",
+		"Allowed behaviors for the tactic command, example: " QQ("default, attack, defend"),
```

Comment:
How about using behavior trees that already exist for the example? `default, reckless, camper`